src/Illuminate/Collections/Collection.php

```diff
@@ -600,7 +603,7 @@ public function has($key)
     {
         $keys = is_array($key) ? $key : func_get_args();
 
-        return array_all($keys, fn ($key) => array_key_exists($key, $this->items));
+        return array_all($keys, fn ($key) => array_key_exists($key ?? '', $this->items));
```

Comment:
Do we want to consider `null` to be `''`? If we have an array containing an item with `''`, it will be returned, when `null` is given, which can be unintentional.

Reply:
Using `null` as key currently has the same behaviour:
```
> $arr[null] = 'test'
= "test"

> $arr
= [
    "" => "test",
  ]

> $arr['']
= "test"
```

Reply:
Thanks for the explanation 👍🏻